api/src/main/java/org/apache/commons/rdf/api/RDFSyntax.java

```diff
@@ -91,7 +148,30 @@
      * Datasets</a>.
      */
     public boolean supportsDataset();
+   
+    /**
+     * Return the RDF 1.1 serialization syntaxes.
+     * <p>
+     * This lists the W3C standardized RDF 1.1 syntaxes like {@link #TURTLE} and
+     * {@link #JSONLD}. Note the existence of other RDF syntaxes that are not
+     * included here, e.g. <a href="http://www.w3.org/TeamSubmission/n3/">N3</a> and
+     * <a href="https://en.wikipedia.org/wiki/TriX_%28syntax%29">TriX</a>.
+     * <p>
+     * The syntaxes returned only support the {@link #mediaType()}
+     * and {@link #fileExtension()} as defined in the corresponding 
+     * W3C specification.
+     * 
+     * @return
+     *      A set of the official RDF 1.1 {@link RDFSyntax}es.
+     * 
+     * @see <a href="https://www.w3.org/TR/rdf11-primer/#section-graph-syntax">RDF
+     *      1.1 Primer</a>
+     * @see org.apache.commons.rdf.experimental.RDFParser
+     */
 
+    public static Set<RDFSyntax> w3cSyntaxes() {
```

Comment:
`W3CRDFSyntax` is package-protected. It was moved it out followingyour advice, but I don't see any win in making it `public` as it has no additional methods (and shouldn't have).

If we make it public and remove `w3cSyntaxes()` , then there's not much more any point to have the proxy constants `RDFSyntax.TURTLE` etc. as they would also then be made public at `W3CRDFSyntax.TURTLE` etc - which - not withstanding the CAPSISSUE - is not as accessible for the 95% use cases of parsing/writing a RDF 1.1 syntax.